### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by compae**

### CODE REVIEW

File: src/main/scala/org/apache/spark/streaming/rabbitmq/ConfigParameters.scala [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/ConfigParameters.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/RabbitMQUtils.scala [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/RabbitMQUtils.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/consumer/Consumer.scala [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/consumer/Consumer.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/consumer/ConsumerParamsUtils.scala [hunk 4]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/consumer/ConsumerParamsUtils.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/distributed/RabbitMQDStream.scala [hunk 5]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/distributed/RabbitMQDStream.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/distributed/RabbitMQDistributedKey.scala [hunk 6]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/distributed/RabbitMQDistributedKey.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/distributed/RabbitMQPartition.scala [hunk 7]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/distributed/RabbitMQPartition.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/distributed/RabbitMQRDD.scala [hunk 8]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/distributed/RabbitMQRDD.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/models/ExchangeAndRouting.scala [hunk 9]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/models/ExchangeAndRouting.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/models/QueueConnectionOpts.scala [hunk 10]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/models/QueueConnectionOpts.scala]: This is the old format

File: src/main/scala/org/apache/spark/streaming/rabbitmq/receiver/RabbitMQInputDStream.scala [hunk 11]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/main/scala/org/apache/spark/streaming/rabbitmq/receiver/RabbitMQInputDStream.scala]: This is the old format

File: src/test/java/org/apache/spark/streaming/rabbitmq/JavaRabbitMQConsumer.java [hunk 12]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/test/java/org/apache/spark/streaming/rabbitmq/JavaRabbitMQConsumer.java]: This is the old format

File: src/test/java/org/apache/spark/streaming/rabbitmq/JavaRabbitMQDistributedConsumer.java [hunk 13]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/test/java/org/apache/spark/streaming/rabbitmq/JavaRabbitMQDistributedConsumer.java]: This is the old format

File: src/test/scala/org/apache/spark/streaming/rabbitmq/RabbitMQConsumerIT.scala [hunk 14]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (danielcsant) [inline src/test/scala/org/apache/spark/streaming/rabbitmq/RabbitMQConsumerIT.scala]: This is the old format

File: src/test/scala/org/apache/spark/streaming/rabbitmq/RabbitMQDistributedConsumerIT.scala [hunk 15]
```diff
@@ -0,0 +1,81 @@
+/**
```
Comment (danielcsant) [inline src/test/scala/org/apache/spark/streaming/rabbitmq/RabbitMQDistributedConsumerIT.scala]: This is the old format

File: pom.xml [hunk 16]
```diff
@@ -96,8 +99,38 @@
             <artifactId>joda-time</artifactId>
             <version>${joda.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.sstone</groupId>
+            <artifactId>amqp-client_${scala.binary.version}</artifactId>
+            <version>1.5</version>
```
Comment (danielcsant) [inline pom.xml]: This dependency is with test scope

**Comment by compae** [comment]
The old format is associated to parent version 0.8.0

**Pull request merged**